pkg/vpc/vpc_test.go

```diff
@@ -58,12 +57,11 @@ var _ = Describe("VPC Endpoints", func() {
 
 			err := UseEndpointAccessFromCluster(p, cfg)
 			if err != nil {
-				Expect(err.Error() == eks.ErrCodeResourceNotFoundException)
```

Comment:
Was this failing before?

Reply:
it's not failing before as it's doing nothing :). Expect(err.Error() == eks.ErrCodeResourceNotFoundException) is just equivalent to Expect(boolean), which is `Assertion` interface https://github.com/onsi/gomega/blob/master/gomega_dsl.go#L150

Reply:
If you just commented this line, the test is still passing

```golang
			if err != nil {
				//Expect(err.Error() == eks.ErrCodeResourceNotFoundException)
			}
            // these 3 lines are doing nothing as well
			Expect(err == nil) 
			Expect(cfg.VPC.ClusterEndpoints.PrivateAccess == &e.Private)
			Expect(cfg.VPC.ClusterEndpoints.PublicAccess == &e.Public)
```